frontend/javascripts/dashboard/dataset/dataset_import_view.js

```diff
@@ -302,13 +302,17 @@ class DatasetImportView extends React.PureComponent<Props, State> {
       this.props.form.setFieldsValue({
         dataSourceJson: jsonStringify(inferredDataSource),
       });
-      this.setState(currentState => {
-        const updatedStatus = {
-          ...currentState.dataSourceSettingsStatus,
-          appliedSuggestions: AppliedSuggestionsEnum.Yes,
-        };
-        return { dataSourceSettingsStatus: updatedStatus };
-      });
+      this.setState(
+        currentState => {
+          const updatedStatus = {
+            ...currentState.dataSourceSettingsStatus,
+            appliedSuggestions: AppliedSuggestionsEnum.Yes,
+          };
+          return { dataSourceSettingsStatus: updatedStatus };
+        },
+        // Enforce validation as antd does not do this automatically.
+        () => this.props.form.validateFields(),
+      );
```

Comment:
I tried to enforce the field validation after line 304 (after updating the values with `setFieldsValue`) but this did not work correctly and I do not know why. I do not think that `setFieldsValue` is asynchronous :/.

Therefore I just added it after the state update (line 314) and this worked fine a few times I tried. Can you think of a better option or do you think that's alright @daniel-wer?

Reply:
Hmm that's strange. By "this did not work correctly" - do you mean that the validation failed or was there some other error?

Reply:
The validation did succeed without an error, but there should have been an error as the default value of the bounding boxes that was used is `0,0,0,0,0,0`.